### PR TITLE
Hybrid tutorial

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jobergum** [comment]
@andreer since you have some experience with hybrid search questions :) I'll ask you to review, this is no way perfect, but maybe a start.

**Comment by andreer** [comment]
This is great. A few comments: 

It would be good to include a little bit of motivation in the start - "we will later see that using hybrid search to combine both methods leads to better results than using only embeddings or text matching" or something

python module "requests" is required but not installed (I'm sure most have it but still)

> the content cluster capacity can be increased by adding node elements ...
move this down to nodes?

services.xml is missing root tag <services>, and "__main__" becomes just "main", at least here in the github markdown preview - make sure this becomes correctly copy-pastable in the doc

I suggest sending this also to review by someone new to search/hybrid search like @esolitos :-)

**Review by andreer — APPROVED** [review]

**Review by esolitos — APPROVED** [review]
I am not extremely familiar yet with all the terminology and relevancy algos, but I was able to follow the tutorial.

I left a couple of comments here and there, but this looks good. 😊 

I noticed some slightly different relevancy scores in some commands, but I am not sure if that is relevant.

File: en/tutorials/hybrid-search.md [hunk 1]
```diff
@@ -0,0 +1,1073 @@
...
+
+
+## Sample queries 
+We can now run a few sample queries to demonstrate various ways to perform searches over this data using the [Vespa query language](../query-language.html).
+
+<div class="pre-parent">
+  <button class="d-icon d-duplicate pre-copy-button" onclick="copyPreContent(this)"></button>
+<pre data-test="exec" data-test-assert-contains="PLAIN-2">
+$ ir_datasets export beir/nfcorpus/test queries | head -1
+</pre>
```
[the remark this answers is not in the document]
Comment (jobergum) [reply en/tutorials/hybrid-search.md:357]: Thanks for this, changed it to `ir_datasets export beir/nfcorpus/test queries --fields query_id text |head -1`

File: en/tutorials/hybrid-search.md [hunk 2]
```diff
@@ -0,0 +1,1073 @@
...
+### Define our first simple hybrid rank profile
+
+First, we can add our first simple hybrid rank profile that combines the dense and sparse components using multiplication to 
+combine them into a single score. 
+
+<pre>
+closeness(field, embedding) * (1 + bm25(title) + bm25(text))
+</pre>
+
+- the `closeness(field, embeddding)` rank-feature returns a score in the range 0 to 1 inclusive
```
[the remark this answers is not in the document]
Comment (jobergum) [reply en/tutorials/hybrid-search.md]: Thanks, added the link in a rewrite

File: en/tutorials/hybrid-search.md [hunk 3]
```diff
@@ -0,0 +1,1073 @@
...
+Notice that we choose `ranking` to specify which rank profile to rank the documents retrieved by the query. 
+This query returns the following [JSON result response](../reference/default-result-format.html):
+
+<pre>{% highlight json %}
+{
+    "root": {
+        "id": "toplevel",
+        "relevance": 1.0,
+        "fields": {
+            "totalCount": 65
```
[the remark this answers is not in the document]
Comment (jobergum) [reply en/tutorials/hybrid-search.md:395]: Will investigate this. Thanks for highlighting and running through it!

**Pull request merged**